reference/docs-conceptual/PowerShell-Support-Lifecycle.md

```diff
@@ -95,16 +109,17 @@ experimentation and feedback.
 > [!NOTE]
 > CIM, PowerShell Remoting, and DSC are not supported on Alpine.
 
-## PowerShell releases end-of-life
+## PowerShell releases end of life
 
-Based on [Lifecycle of PowerShell Core](#lifecycle-of-powershell-core), the following table lists
+Based on [Lifecycle of PowerShell](#lifecycle-of-powershell-7), the following table lists
 the dates when various releases will no longer be supported.
 
-| Version | End-of-life                   |
-|---------|-------------------------------|
-| 6.0     | February 13, 2019             |
-| 6.1     | September 28, 2019            |
-| 6.2     | 6 months after 7 releases     |
+| Version |    End-of-life     |
+| :-----: | ------------------ |
+|   7.0   | December 3, 2022   |
+|   6.2   | September 4, 2020  |
+|   6.1   | September 28, 2019 |
```

Comment:
Since these dates have passed, do we need to include them?

Also, is 6.2 supported for 1 year since it was released under the Modern Lifecycle which requires a 12-month notice window?

Reply:
Yes, we should include them for historical reasons. Microsoft's overall policy website and .NET's policy page do the same. 

And no, [the terms](https://support.microsoft.com/en-us/help/30881/modern-lifecycle-policy) read as such, and we've provided a successor with PS 7:

> 12 months' notification prior to ending support if no successor product or service is offered